stock_batch_picking/wizard/batch_picking_creator.py

```diff
@@ -27,7 +46,7 @@ class StockBatchPickingCreator(models.TransientModel):
 
     picker_id = fields.Many2one(
         'res.users', string='Picker',
-        default=lambda self: self.env.user.company_id.default_picker_id,
+        default=_default_picker_id,
```

Comment:
Please utilize lambda declaration for defaults in order to allow for overloading via inherited models & for proper method/field ordering: `lambda s: s.default_picker_id()`